README.rst

```diff
@@ -197,17 +197,11 @@ Review the vars defined within `ansible/vars/defaults.yml`. In here you can cust
 However, make sure to define `ssh_key_path` to point to the location of the SSH key in use amongst the nodes and also `vxlan_vni` which should be unique value between 1 to 100,000.
 VNI should be much smaller than the officially supported limit of 16,777,215 as we encounter errors when attempting to bring interfaces up that use a high VNI. You must set``vault_password_path``; this should be set to the path to a file containing the Ansible vault password.
 
-Finally run the ansible playbooks.
-You may need to run `fix-homedir-ownership.yml` if you are using an image that has `ansible_user` not owning their own home folder.
-You may also need to run `grow-control-host.yml` if you are using LVM images and the LVMs are too small to install Ansible.
-If not you can skip those playbook and proceed onto `deploy-openstack-config` which shall configure your Ansible control host in preparation for deployment.
+Finally, run the configure-hosts playbook.
```

Comment:
I'm not sure whether I should remove the description here completely, since it does raise the valid point that you can sometimes save time by skipping playbooks. I don't know if it's worth the extra complexity though